libraries/pom.xml

```diff
@@ -8,13 +8,13 @@
     </parent>
 
     <artifactId>libraries</artifactId>
+    <version>2.0.2-SNAPSHOT</version>
     <packaging>pom</packaging>
 
     <name>(Comcast) ${project.artifactId}</name>
     <description>DAWG library</description>
 
     <modules>
-        <module>dawg-config</module>
```

Comment:
But the default information is pretty useless, it just sets things to localhost.